nanovdb/nanovdb/python/PyCreateNanoGrid.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
```

Comment:
Should these headers use `#ifdef` guard instead of `#pragma once` to be inline with the header style?

Reply:
I'm 50/50. The examples in NanoVDB use `#pragma once` which is more concise, less error prone, and arguably the more modern solution.

Reply:
Yeah, it's just that in the OpenVDB coding style guide it mentions using `#ifdef` guards.  We could change it if people think we should move that way, I had to change all of fVDB to move to the guards

https://www.openvdb.org/documentation/doxygen/codingStyle.html#sIncludeStatements

Reply:
Changed

Reply:
Cool, thanks!